bin/fetch_windows_zones.php

```diff
@@ -44,6 +43,6 @@
 
 echo "Formatting\n";
 
-exec(__DIR__.'/sabre-cs-fixer fix '.escapeshellarg($outputFile));
+exec(__DIR__.'/../vendor/bin/php-cs-fixer fix '.escapeshellarg($outputFile));
```

Comment:
hmm not sure its correct that this path changed? could you verify this bin still works?

Reply:
I ran `fetch_windows_zones.php` locally from the root of the repo in order to update the windows zones file.
That is how I found that the old `sabre-cs-fixer` is no longer, and made it work by doing this.